Sources/ApolloCodegenLib/ApolloCodegenOptions.swift

```diff
@@ -169,8 +179,13 @@ public struct ApolloCodegenOptions {
       arguments.append("--omitDeprecatedEnumCases")
     }
     
-    if self.passthroughCustomScalars {
+    switch customScalarFormat {
+    case .default:
+      break
+    case .passthrough:
       arguments.append("--passthroughCustomScalars")
+    case .passthroughWithPrefix(let prefix):
+      arguments.append("--customScalarsPrefix='\(prefix)'")
```

Comment:
I think this needs to have both `--passthroughCustomScalars` and `--customScalarPrefix`, doesn't it?

Reply:
I took a look at Apoll-tooling and it sets `passthroughCustomScalars` if `customScalarsPrefix` is set.
https://github.com/apollographql/apollo-tooling/blob/aed906a0f29f9e7f801484551c21e1793f6cb0d0/packages/apollo/src/commands/client/codegen.ts#L214

I also ran it manually and the behaviour matched. I can add the flag back in if you want it to be explicit.

Reply:
[It's not guaranteed in the README](https://github.com/apollographql/apollo-tooling#apollo-clientcodegen-output) and I don't see anyplace where that's being double-checked in the tests - I'd rather go belt-and-suspenders here for now